adal/src/main/java/com/microsoft/aad/adal/AuthenticationResult.java

```diff
@@ -69,55 +69,55 @@
         Succeeded,
     }
 
-    private String mCode;
+    protected String mCode;
```

Comment:
I'm not seeing usages in the code that explain why this became `protected` -- is this still needed?

Reply:
this variable is has no other usage. I've changed it back to private. But other variables needs to be updated to 'protected' since BrokerTokenResult in Ad-accounts is extending AuthenticationResult and will use the member variables directly.